src/chart/components/y_axis/price_labels/price-label.drawer.ts

```diff
@@ -96,22 +96,30 @@ export function drawLabel(
 		showLine && avoidAntialiasing(ctx, () => drawLine(ctx, lineXStart, lineY, lineXEnd, lineY, 1));
 	const _drawLabel = () => drawLabel(ctx, bounds, text, centralY, visualLabel, config, colors, false, backgroundCtx);
 
-	switch (mode) {
-		case 'line':
-			_drawLine();
-			_drawDescription();
-			break;
-		case 'line-label':
-			_drawLine();
-			_drawDescription();
-			_drawLabel();
-			break;
-		case 'label':
-			_drawDescription();
-			_drawLabel();
-			break;
-		case 'none':
-			break;
+	const drawLineLabel = () => {
```

Comment:
this kind of refactoring creates additional context/resources usage for V8, like functions creation, Object creation and then lookup the memory for given drawer. Why don't just leave at as it was right here? I guess it's not necessary to have such an abstraction here. You can just early return with `ctx.restore` right before the `switch` and that's all